cves/CVE-2016-2048.yml

```diff
@@ -173,20 +198,25 @@ subsystem:
     In the name field, a subsystem name (or an array of names)
 
     e.g. clipboard, model, view, controller, mod_dav, ui, authentication
-  answer:
-  name:
+  answer: 
+    This error occured in the django/contrib/admin package,
+    as part of options handling for ModelAdmin.
+  name: Admin
 interesting_commits:
   question: |
     Are there any interesting commits between your VCC(s) and fix(es)?
 
     Write a brief (under 100 words) description of why you think this commit was
     interesting in light of the lessons learned from this vulnerability. Any
     emerging themes?
+  answer:
+    I did not find any commits relating to this vulnerability, 
+    though one might consider my first VCC something of an interesting commit,
+    as it set the stage for the poor maintainability of that add permissions-check,
+    which undoubtedly asisted the second VCC I listed.
```

Comment:
If you found no interesting commits, just put in a brief summary of some of the commits in the meantime to show you went through them.

Reply:
Fixed.